swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.item import Item
+class Clothing(Item):
+    pass
```

Comment:
`pass` is a Python placeholder to avoid syntax errors before code is added.  Once code is added `pass` can be removed.